lua/entities/gmod_wire_vectorthruster.lua

```diff
@@ -246,11 +250,9 @@ function ENT:TriggerInput(iname, value)
 end
 
 function ENT:PhysicsSimulate( phys, deltatime )
-	if self.calcforce then
-		self:CalcForce(phys)
```

Comment:
Should revert the self.calcforce changes and instead set self.calcforce to false in this if

Reply:
The force is actually global so nvm